### PR TITLE
[C#] Extract concepts from v2 exercise two-fer

### PR DESCRIPTION
**Pull request by fpetru**
work in progress, ref #624

### CODE REVIEW

**Comment by fpetru** [comment]
please help to review this PR. thank you

File: languages/csharp/reference/exercise-concepts/two-fer.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Concepts of Two Fer
+
+[Example implementation](https://github.com/exercism/csharp/blob/master/exercises/two-fer/Example.cs)
+
+## General
```
Comment (ErikSchierboom) [inline languages/csharp/reference/exercise-concepts/two-fer.md]: This is great! The only things that I would add are the "strings" and "function arguments" (of which optional arguments is a specialized Concept) Concepts. Does that make sense?

Comment (fpetru) [reply languages/csharp/reference/exercise-concepts/two-fer.md]: Sure, thank you for your review. I will add these.

**Comment by ErikSchierboom** [comment]
Thanks @fpetru!

**Pull request merged**